dashboard/config/scripts/levels/allthethings_fish_tank.level

```diff
@@ -46,7 +46,7 @@
     "parent_level_id": 12950,
     "auto_run_setup": "DRAW_LOOP",
     "show_type_hints": "true",
-    "validation_code": "if (World.frameCount > 50) {\r\n  var status = 0;\r\n  \r\n  // Check through all sprites in case student changed the sprite name\r\n  for (var i=0; i<World.allSprites.length; i++) {\r\n    var sprite = World.allSprites[i];\r\n    // ignore edge sprites\r\n    if (!edges.contains(sprite)) {\r\n      if (sprite.getAnimationLabel() == \"tumbleweed\") {\r\n        status = 3;\r\n      }\r\n    }\r\n  }\r\n  levelSuccess(status);\r\n}",
+    "validation_code": "if (World.frameCount > 50) {\r\n  var status = 0;\r\n  var animations = getAnimationsInUse();\r\n  for (var i = 0; i < animations.length; i++) {\r\n    if (animations[i] == \"tumbleweed\") {\r\n      status = 3;\r\n    }\r\n  }\r\n  levelSuccess(status);\r\n}",
```

Comment:
Still not sure I understand the original failure mode here. The previous iteration threw because `edges` was undefined during preview?

Reply:
With the original change, the edge sprites are only created when they are needed (when the `isTouchingEdges` or `edgesDisplace` blocks are used). The allthethings level doesn't use either of these blocks, so it shouldn't care about `edges`, but the validation code was referencing `edges` directly, which broke because `edges` won't be defined anymore. Sorry not sure I'm explaining that adequately...

Reply:
With this change, runtime vs preview won't matter, but `edges` will be undefined until the first time `isTouchingEdges` or `edgesDisplace` is called.

Reply:
Never mind, I should've read the PR description more carefully!

I do wonder if preview code should be creating the edge sprites, for consistency?